matrix-sdk/src/main/java/org/matrix/androidsdk/util/EventDisplay.java

```diff
@@ -47,20 +49,33 @@
 public class EventDisplay {
     private static final String LOG_TAG = EventDisplay.class.getSimpleName();
 
+    private static final String MESSAGE_IN_REPLY_TO_FIRST_PART = "<blockquote data-mx-reply>";
```

Comment:
Heads up that we made a last minute change to the spec here, to simplify things a bit - sorry :(  See https://github.com/matrix-org/matrix-react-sdk/pull/1889

Reply:
Ok, is it deployed yet?

Reply:
replies are a client-side feature, and as per that PR the format change is already merged and so present on riot.im/develop and will also be in the forthcoming 0.15 release later today (albeit disabled by default in labs).

Reply:
ok, fixed